src/Dialog/formComponents/widgets/CodeWidget.jsx

```diff
@@ -1,26 +1,53 @@
-import React from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import CodeMirror from 'react-codemirror';
+import jsyaml from 'js-yaml';
 import 'codemirror/mode/yaml/yaml';
 import 'codemirror/mode/javascript/javascript';
 
-function CodeWidget(props) {
-  const {readonly, value, onChange} = props;
-  const {format} = props.schema;
+class CodeWidget extends Component {
+  componentDidMount() {
+    var that = this;
+    setTimeout(() => {
+      that.editor.getCodeMirror().refresh();
```

Comment:
Arrow function should be enough, no need to pass extra context :)

Reply:
Done